main.py

```diff
@@ -12,10 +12,14 @@ def main():
     # A função abaixo abre o arquivo texto numeros.txt em modo leitura e lê as linhas dele separando
     # elas em uma lista de strings.
     with open('números.txt', 'r', encoding='utf8') as arquivo:
-        linhas_do_arquivo = arquivo.readlines()
+      linhas_do_arquivo = arquivo.readlines()
 
     # Pegue a lista de strings e converta todos os valores dentro dela para inteiros.
-    # TODO COLOQUE SEU CÓDIGO AQUI E APAGUE ESSE COMENTÁRIO DEPOIS.
+
+    for lis in linhas_do_arquivo:
+      linhas_do_arquivo.append(int(lis))
```

Comment:
Um jeito mais elegante de fazer isso é:
```suggestion
    linhas_do_arquivo = map(int, linhas_do_arquivo)
```